PhysicsTools/PatAlgos/python/slimming/modifiedElectrons_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
How do I keep the originally named ones from running in unscheduled mode? del them in python?